### PR TITLE
1.5.10

### PR DESCRIPTION
**Pull request by GeekPress**

### CODE REVIEW

File: inc/functions/admin.php [hunk 1]
```diff
@@ -161,4 +161,30 @@ function imagify_query_results_combine( $keys, $values ) {
...
+ *
+ * @since  1.5.10
+ * @author Jonathan Buttigieg
+ *
+ * @return int The buffer size
+ */
+function get_imagify_bulk_buffer_size() {
+	$sizes = count( get_imagify_thumbnail_sizes() );
+	
+	switch (true) {
```
Comment (Tabrisrp) [inline inc/functions/admin.php]: spaces in the brackets for coding standard

File: readme.txt [hunk 2]
```diff
@@ -135,11 +135,14 @@ When the plugin is disabled, your existing images remain optimized. Backups of t
...
 == Changelog ==
+= 1.5.10 =
+* Improvement
+ * Set to 1 the Bulk buffer size when there are more than 10 thumbnails to avoid "Unkown error" on the Bulk Optimization
+ 
 = 1.5.9 =
 * Bug fix
  * Don't delete the thumbnail when the maximum file size is set to one of the thumbnail size
  * Don't strip the image meta data if possible (only with Imagick)
- * Fix persistent "WELL DONE" message because of "original_size" meta value was 0
```
Comment (Tabrisrp) [inline readme.txt]: This was removed on purpose?

Comment (GeekPress) [inline readme.txt]: @Tabrisrp It's a mistake, I will re-add it. Thanks

**Comment by GeekPress** [comment]
@Tabrisrp the 2 comments are fixed.

**Pull request merged**